src/styles/_datavisyn.scss

```diff
@@ -28,24 +28,33 @@ $dataviyn_font: Monda !default;
 
 nav.navbar,
 nav.navbar-inverse {
+  min-height: auto; // Temp fix until https://github.com/phovea/phovea_clue/issues/140 is fixed
+
   .navbar-brand.caleydo_app {
     background: none;
     padding-left: 0;
+    display: flex;
+    align-items: center;
 
     &:before {
       @include datavisyn-font();
-      content: 'd';
-      font-size: 80%;
-      background: black;
-      border-radius: 5px;
-      padding: 0.2em 0.3em;
-      margin-left: 0.2em;
+      display: inline-block;
+      height:14px;
```

Comment:
I would probably make it 1-2 px smaller so that the d has the same height as the title. But we can try that tomorrow at your screen.

Reply:
Btw. there is a space missing after the `:`